### PR TITLE
0.6.0 rc rebase into main

### PR DESCRIPTION
**Pull request by NickGeneva**
# Modulus Pull Request

## Checklist

- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by mnabian — APPROVED** [review]

File: CITATION.cff [hunk 1]
```diff
@@ -1,6 +1,7 @@
-cff-version: 0.5.0
+cff-version: 1.2.0
```
Comment (NickGeneva) [inline CITATION.cff:1]: This was incorrect, should be schema version here
https://github.com/citation-file-format/citation-file-format

Comment (NickGeneva) [reply CITATION.cff:1]: Added version field below :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,29 @@ All notable changes to this project will be documented in this file.
...
 
-## [0.6.0a0] - 2024-04-XX
+## [0.7.0a0] - 2024-07-XX
+
+### Added
+
+- Code logging for CorrDiff via wandb.
+- Augmentation pipeline for CorrDiff.
+- Regression output as additional conditioning for CorrDiff.
+- Leanable positional embedding for CorrDiff.
```
Comment (Alexey-Kamenev) [inline CHANGELOG.md:16]: Leanable -> Lea**r**nable?

**Review by Alexey-Kamenev — APPROVED** [review]
LGTM

**Pull request merged**